state.json

```
@@ -233,7 +233,9 @@
                 "name": "pocket",
                 "random": true,
                 "commands": [
-                    "gslide id=1gDlOjkeMb9xLHzxhb5P5XL7gpIq08w0hdjCdykRIGTY comment=\"Pocket slideshow\""
+                    "gslide id=1gDlOjkeMb9xLHzxhb5P5XL7gpIq08w0hdjCdykRIGTY comment=\"Pocket slideshow\"",
+                    "https://forecast.io/embed/#lat=37.779&lon=-122.41499&name=Pocket%20-%20SanFrancisco&color=#DC143C&font=Arial zoom=300 comment=\"SF Weather\"",
+                    "<iframe src='https://modeanalytics.com/getpocket/reports/13a9c899c073/runs/1368ea3a7fd2/embed' width='100%' height='300' frameborder='0'></iframe>"
```

We ould use HTML like this, but it would have to be a slightly different format. That being said, using the URL directly would work better in this case, like this:

```suggestion
                    "https://modeanalytics.com/getpocket/reports/13a9c899c073/runs/1368ea3a7fd2/embed"
```

That would pass the URL to Corsica, which would use it as an iframe src.

That being said, that page doesn't load for me, it gives me a "Report not found, you don't have permissions" error. I think our next best bet will be to write a small page that uses the JS API to show a graph with the correct credentials. Or we could try and find a way to embed credentials in this so that it works, or remove that frame header from the original idea.

---

I will remove this link for now, and work on figuring out a solution later. Thanks for the help!